CMakeLists.txt

```diff
@@ -52,7 +52,8 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CPP_STANDARD_FLAGS}")
 
 
 #--- Declare options -----------------------------------------------------------
-option(CREATE_DOC "Whether or not to create doxygen doc target.")
+option(CREATE_DOC "Whether or not to create doxygen doc target." OFF)
+option(BUILD_TESTS "Whether or not to build unit and integration tests." ON)
```

Comment:
```
include(CTest)
```
already defines an option `BUILD_TESTING`, which should be used for this purpose

Reply:
Wasn't aware of this variable, that is better indeed, thanks.